Making updates to the SEO best practices page (#12711)
PostHog/posthog.com: contents/handbook/content/seo-guide.md

sidebar: Handbook
showTitle: true
---
## General principles

### 1. Start with search intent

Don't obsess over exact-match keywords, ask: What is the searcher really trying to accomplish? For example, a user searching "difference between retention rate and churn" will likely also benefit from actionable insight on improving customer retention, not just definitions.

We craft our content to address those underlying needs. Cover the main topic thoroughly, include related sub-questions and themes, and anticipate next steps a reader might take.

Here’s what that might look like for "Retention rate vs churn"

- **Quick answer first:** Retention rate shows who stayed, churn shows who left. If churn is 20%, retention is 80%.  
- **Context next:** Add formulas, a simple numeric example, and a short paragraph on why retention matters for growth.  
- **Related questions:**  
  - What’s a "good" churn rate?  
  - How do you reduce churn? 
  - When should you focus on retention vs. acquisition?  
- **Next steps:** Link to guides on retention strategies, cohort analysis, and churn reduction.

### 2. Make it easy to digest

> When answering a question, we lead with the answer first, then expand with supporting details. This helps impatient readers and aligns with how AI tools select responses.

Reviewer: ```suggestion
When answering a question, lead with the answer first, then expand with supporting details. This helps impatient readers and aligns with how AI tools select responses.
```

We should be bold, creative, and opinionated – but not so clever that we lose relevance. If every result has a nearly identical headline, we win by being different, but if we get too abstract or too witty, we risk missing the actual query intent and dropping out of search entirely.

> **Quick rule of thumb:** If it sounds like every other search result, sharpen it. If it sounds clever but hides what the article’s about, clarify it. 

### 4. Demonstrate expertise and authority

The internet has never been so full of words. The friction for content creation has dropped to nearly zero (thanks, ChatGPT), which means the bar for quality has shot up. The only way to win attention is to raise the bar: create content that actually teaches, clarifies, and adds something new.

Establish yourself (and PostHog) as a subject matter expert. We do this by:

- **Backing our claims with data.** Include relevant statistics, research findings, or mention credible studies. Citing reputable sources or adding footnotes for facts can also build trust (and AI models tend to favor answers with a cited source).

> - **Including expert insights.** If possible, add quotations or insights from experts (it can be an internal SME). An authoritative quote or a first-hand insight provides uniqueness and value that generic content lacks.

Reviewer: +1: just as a general comment, we should avoid jargon and abbreviations as much as possible.

For example, the parent search term "user persona" (27,000/mo) has numerous derivations:

- Define user persona (8,100)  
- Create user persona (3,600)  
- Persona modelling (720)  
- Benefits of personas (50)  
- User persona examples (5,400)  
- Examples of user persona (260)  
- How to create personas (2,900)  
- What is a user persona (260)  
- User persona template (27,000)

We target clusters of intent, not just one keyword. Long-tail variations are often easier wins and build topical relevance. Over time, our page can rank for multiple terms and even capture the broad head term as authority grows.

### 7. Write for our ICP

The more specific we make our content, the more likely it is to resonate – and perform. This matters more than ever with AI-driven search and tools like ChatGPT's Deep Research, which don’t just answer the initial query but often fan out into follow-up questions and related recommendations. 

> For example, a generic “best CRM tools” list might compete with thousands of others. But “best CRM tool for a healthtech startup in 2025” positions us as the exact match for a highly qualified search.

Reviewer: +1. Also, I'd generally discourage adding "in 2025" into our titles. Starts to feels a bit derivative doing that, even if it's technically good practice.

> For example, a generic “best CRM tools” list might compete with thousands of others. But “best CRM tool for a healthtech startup in 2025” positions us as the exact match for a highly qualified search.

Author: Will update the example! RE: adding "in 2025", I can agree with us not adding it to headlines but I think a good compromise would be adding it to meta titles. The downside here means having to go back and update them in bulk once a year (maybe there's an elegant solution we can think of down the line to allow for bulk metadata updates), but it signals freshness for search engines and LLMs which is becoming increasingly more relevant

- **Chunk quality:** Keep sections focused. A 200-word section that completely answers one question is more reusable than a 1,000-word wall of text.  
- **Stay fresh and correct:** Outdated or wrong info can keep us out of results (or worse, get us quoted incorrectly). Include timestamps, years, and up-to-date references (“as of 2025”).  
- **Favor Q&A format:** Perplexity loves conversational answers and listicles (think “Top 5 tools for X”).  
- **Consistency matters:** Keep facts about PostHog accurate and aligned across different pieces of content.  
- **Watch competitors:** Monitor what SGE cites and improve on those answers to outrank them.

### Additional tips

Good metadata is like a handshake – it’s the first impression users (and AI tools) get before they ever see the page. Well-crafted titles and descriptions can improve click-through rates and help AI engines understand context.

Quick metadata checklist:

- [ ] Meta title includes the primary keyword and stays under \~60 characters  
- [ ] Meta description is under 160 characters and compelling (can include primary or secondary keywords)  
- [ ] Each page has unique metadata (no duplicates)  

> - [ ] Preview in a SERP simulator before publishing to check for truncation  

Reviewer: ```suggestion
- [ ] Preview in a [SERP simulator](https://mangools.com/free-seo-tools/serp-simulator) before publishing to check for truncation  
```

> - [ ] Preview in a SERP simulator before publishing to check for truncation  

Reviewer: Added a suggestion

- [ ] Add dates, numbers, or benefit-driven language where relevant to make metadata feel fresh and worth clicking.

> 

Reviewer: This does exist and I've used it a handful of times. Probably not urgent right now, but we could with re-organizing a lof the info around frontmatter + components as it's scattered all over the place and hard to find atm.

> 

Author: Closing this for now since the documentation around metadata specifically doesn't exist but I can update it with a link once we clean that up in the handbook

For example, the parent search term "user persona" (27,000/mo) has numerous derivations:

- Define user persona (8,100)  
- Create user persona (3,600)  
- Persona modelling (720)  
- Benefits of personas (50)  
- User persona examples (5,400)  
- Examples of user persona (260)  
- How to create personas (2,900)  
- What is a user persona (260)  
- User persona template (27,000)

We target clusters of intent, not just one keyword. Long-tail variations are often easier wins and build topical relevance. Over time, our page can rank for multiple terms and even capture the broad head term as authority grows.

### 7. Write for our ICP

The more specific we make our content, the more likely it is to resonate – and perform. This matters more than ever with AI-driven search and tools like ChatGPT's Deep Research, which don’t just answer the initial query but often fan out into follow-up questions and related recommendations. 

> For example, a generic “best CRM tools” list might compete with thousands of others. But “best CRM tool for a healthtech startup in 2025” positions us as the exact match for a highly qualified search.

Author: ```suggestion
For example, a generic “Best session replay tools” list might compete with thousands of others. But “Best open source session replay tools for startups” positions us as the exact match for a highly qualified search.
```
